### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by LukasSkywalker** [abraunegg/onedrive#1707] [General]
I have been using onedrive on Linux for a few years now and it has served me very well. Thank you so much for creating this client. It is constantly running on my workstation and synchronizing files from three different business and education accounts. I think that is not even possible with the official client on Windows.
I sadly don't really have free time to help out with development, but is there a way I can donate to the project (using Github Sponsors or something similar)?

**Comment by abraunegg** [comment]
@LukasSkywalker 
It took a while to get back to this due to waiting on GitHub to approve enabling sponsorship|donations - so this is now possible

**Reply by kayzzen01** [reply]
@abraunegg, I go through practically the same reasons as the message in the topic..
In order to facilitate donations, do you see the possibility of receiving donations through cryptocurrency such as LiteCoin LTC where transfer costs are cheap?

**Reply by abraunegg** [reply]
Unfortunately, the only way to facilitate donations is via the GitHub sponsorship:

![image](https://github.com/abraunegg/onedrive/assets/4956234/918d818e-838b-4169-94ef-323e1a08fec3)

No crypto donation is possible via that method - sorry.
